.github/workflows/build.yml

```diff
@@ -16,6 +16,11 @@ jobs:
         node-version: '16.9.1'
     - run: npm install npm -g
     - run: npm install
-    - run: node_modules/.bin/electron-builder build
+    - name: Build MacOS Intel
+      run: node_modules/.bin/electron-builder build -m --x64
+      env:
+        GH_TOKEN: ${{ secrets.github_token }}
+    - name: Build MacOS M1/ARM
+      run: node_modules/.bin/electron-builder build -m --arm64
```

Comment:
I needed to explicitly specify `--arm64` to force it to build the M1 version. The GitHub runners are x64/intel.

I'm not sure if there's a "better" way to do this with `electron-builder`.

Reply:
You should be able to put both arguments together in `package.json`, here's example from documentation:

```
"build": {
  "win": {
    "target": [
      {
        "target": "nsis",
        "arch": [
          "x64",
          "ia32"
        ]
      }
    ]
  }
}
```

Reply:
Thanks for the pointer - fixed https://github.com/hovancik/stretchly/pull/1099/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519